src/commands/admin.ts

```diff
@@ -55,6 +55,115 @@ export default new Command({
                             $.channel.send(`The custom prefix for this guild is now \`${$.args[0]}\`.`);
                         }
                     })
+                }),
+                welcome: new Command({
```

Comment:
Actually... shouldn't we have a `permission` entry to give server admins the ability to set this?

Reply:
The permission for `admin set welcome <...>` is inherited from `admin set`, so the command is available for server admins and up.

Reply:
Got it. In that case, it's ready to merge.